dhcp-relay/dhcp_kern_xdp.c

```diff
@@ -136,10 +228,10 @@ static __always_inline int write_dhcp_option_255(void *ctx, int offset) {
 
 /* Calculates the IP checksum */
 static __always_inline int calc_ip_csum(struct iphdr *oldip, struct iphdr *ip,
-	__u32 oldcsum) {
+		__u32 oldcsum) {
```

Comment:
Some odd alignment stuff going on here; generally, for multi-line function definitions, align the continuation of the parameters with the opening parenthesis on subsequent lines.